solidity/contracts/isms/MultisigIsm.sol

```diff
@@ -82,33 +82,26 @@ contract MultisigIsm is IMultisigIsm, Ownable {
     event CommitmentUpdated(uint32 domain, bytes32 commitment);
 
     // ============ Constructor ============
+    struct DomainConfig {
+        uint32 domain;
+        uint8 threshold;
+        address[] validators;
+    }
 
-    // solhint-disable-next-line no-empty-blocks
-    constructor() Ownable() {}
-
-    // ============ External Functions ============
-
-    /**
-     * @notice Enrolls multiple validators into a validator set.
-     * @dev Reverts if `_validator` is already in the validator set.
-     * @param _domains The remote domains of the validator sets.
-     * @param _validators The validators to add to the validator sets.
-     * @dev _validators[i] are the validators to enroll for _domains[i].
-     */
-    function enrollValidators(
```

Comment:
Wouldn't this removal mean that you can't enroll validators afterwords by the owner?

Reply:
happy to leave these
basically didn't like the API of nested arrays

Reply:
Man you really hate nested arrays don't you

Reply:
I don't mind the struct change but making the MultisigIsm validator configs immutable post-initialization is something that merits more discussion

Reply:
they're not immutable, I just got rid of the batch functions